packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -457,6 +457,70 @@ describe('Launchpad: Setup Project', () => {
 
         cy.contains('Project Setup')
       })
+
+      it('can move forward to choose browser if e2e is configured and is selected from the dropdown list', () => {
+        cy.openProject('pristine-with-e2e-testing')
+        cy.visitLaunchpad()
+
+        verifyWelcomePage({ e2eIsConfigured: true, ctIsConfigured: false })
+
+        cy.get('[data-cy-testingtype="e2e"]').within(() => {
+          cy.get('[data-cy=status-badge-menu]').click()
+          cy.get('[data-cy="Launch Browser"]').click()
+        })
+
+        cy.contains('Choose a Browser')
+      })
+
+      it('can reconfigure config from the testing type card selecting E2E', () => {
+        cy.openProject('pristine-with-e2e-testing')
+        cy.visitLaunchpad()
+
+        verifyWelcomePage({ e2eIsConfigured: true, ctIsConfigured: false })
+
+        cy.get('[data-cy-testingtype="component"]').within(() => {
+          cy.contains('Not Configured')
+        })
+
+        cy.get('[data-cy-testingtype="e2e"]').within(() => {
+          cy.get('[data-cy=status-badge-menu]').click()
+          cy.get('[data-cy="Reconfigure"]').click()
+        })
+
+        cy.contains('Project Setup')
+      })
+
+      it('can move forward to choose browser if component is configured and is selected from the dropdown list', () => {
+        cy.openProject('pristine-with-ct-testing')
+        cy.visitLaunchpad()
+
+        verifyWelcomePage({ e2eIsConfigured: false, ctIsConfigured: true })
+
+        cy.get('[data-cy-testingtype="component"]').within(() => {
+          cy.get('[data-cy=status-badge-menu]').click()
+          cy.get('[data-cy="Launch Browser"]').click()
+        })
+
+        cy.contains('Choose a Browser')
```

Comment:
Do this return the user to the launchpad to select a browser? If so, the verbiage `Launch Browser` button is mislead and seems to indicate this behavior is a bug.